### PR TITLE
zn-CN: tutorials/estimators/boosted_trees.ipynb

### PR DESCRIPTION
**Pull request by Berrywrq**

### CODE REVIEW

File: site/zh-cn/beta/tutorials/estimators/boosted_trees.ipynb [hunk 1]
```diff
@@ -0,0 +1,1112 @@
...
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://tensorflow.google.cn/beta/tutorials/estimators/boosted_trees\"><img src=\"https://tensorflow.google.cn/images/tf_logo_32px.png\" />在 tensorFlow.google.cn 上查看</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/estimators/boosted_trees.ipynb\"><img src=\"https://tensorflow.google.cn/images/colab_logo_32px.png\" />在 Google Colab 中运行</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/estimators/boosted_trees.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\">在 GitHub 上查看源代码b</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/estimators/boosted_trees.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/estimators/boosted_trees.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\">在 GitHub 上查看源代码</a>\n",
```

File: site/zh-cn/beta/tutorials/estimators/boosted_trees.ipynb [hunk 2]
```diff
@@ -0,0 +1,1112 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "dW3r7qVxzqN5"
+   },
+   "source": [
+    "本教程是使用基于 `tf.estimator`API的决策树来训练梯度提升模型的端到端演示。提升树（Boosted Trees）模型是回归和分类问题中最受欢迎并最有效的机器学习方法之一。这是一种融合技术，它结合了几个（10 个，100 个或者甚至 1000 个）树模型的预测值。\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/estimators/boosted_trees.ipynb]:
```suggestion
    "本教程是使用基于 `tf.estimator` API的决策树来训练梯度提升模型的端到端演示。提升树（Boosted Trees）模型是回归和分类问题中最受欢迎并最有效的机器学习方法之一。这是一种融合技术，它结合了几个（10 个，100 个或者甚至 1000 个）树模型的预测值。\n",
```

File: site/zh-cn/beta/tutorials/estimators/premade_estimators.ipynb [hunk 3]
```diff
@@ -0,0 +1,610 @@
...
+      "metadata": {
+        "colab_type": "code",
+        "id": "qnf4o2V5lcPn",
+        "colab": {}
+      },
+      "source": [
+        "# 构建一个拥有两个隐层，隐藏节点分别为 30 和 10 的深度神经网络。\n",
+        "classifier = tf.estimator.DNNClassifier(\n",
+        "    feature_columns=my_feature_columns,\n",
+        "    # 隐层所含结点数量分别为 30 和 10.\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/estimators/premade_estimators.ipynb]:
```suggestion
        "    # 隐层所含结点数量分别为 30 和 10。\n",
```

Comment (Berrywrq) [reply site/zh-cn/beta/tutorials/estimators/premade_estimators.ipynb]: Thanks， I have made some modifications， please check

**Pull request merged**